cluster-autoscaler/metrics/metrics.go

```diff
@@ -50,6 +50,12 @@ const (
 	LogLongDurationThreshold = 5 * time.Second
 )
 
+const (
+	ScaleDownNodeDeletion      = "scaleDown:nodeDeletion"
+	ScaleDownFindNodesToRemove = "scaleDown:findNodesToRemove"
+	ScaleDownMiscOperations    = "scaleDown:miscOperations"
```

Comment:
Extracted all the currently existing labels I think. Let me know if this makes sense.